.github/workflows/test_fast.yml

```diff
@@ -75,6 +75,7 @@ jobs:
         with:
           name: cylc-run (${{ matrix.os }} py-${{ matrix.python-version }})
           path: ~/cylc-run/
+          include-hidden-files: true
```

Comment:
I tested this and it's ok to add this already even though the option is not yet included in `actions/upload-artifact@v4`